insurancesimulation.py

```diff
@@ -600,6 +611,63 @@ def record_bankruptcy(self):
     def record_unrecovered_claims(self, loss):
         self.cumulative_unrecovered_claims += loss
 
+    def create_network_representation(self):
+        import networkx as nx
```

Comment:
it bugs me that there is an import within a function

Reply:
As it should!